products/BellHybrid/apps/application-bell-settings/models/alarm_settings/AlarmSettingsModel.cpp

```diff
@@ -1,16 +1,14 @@
-// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 #include <common/models/AlarmSettingsModel.hpp>
-
-#include <apps-common/ApplicationCommon.hpp>
 #include <db/SystemSettings.hpp>
 
 namespace app::bell_settings
 {
     namespace
     {
-        static constexpr std::string_view DefaultBrightness{"50.0"};
+        constexpr std::string_view DefaultBrightness{"50.0"};
```

Comment:
Use `=` as you did before eg. `alarmFrontlightOff` ;)

Reply:
Done, I created `validateBrightness` function as in `FrontlightAction.cpp`